### PR TITLE
Add stripe thank you email

### PR DESCRIPTION
**Pull request by jbguardian**
- Include thank you email into Stripe payment flow.
- Remove unnecessary data to compose thank you email JSON

### CODE REVIEW

File: src/main/scala/backend/PaypalBackend.scala [hunk 1]
```diff
@@ -97,7 +97,7 @@ class PaypalBackend(
       payment <- getPaymentFromCapturePaypalPaymentData(capturePaypalPaymentData)
       identityId <- getOrCreateIdentityIdFromEmail(payment.getPayer.getPayerInfo.getEmail)
       _ = trackContribution(payment, capturePaypalPaymentData.acquisitionData, identityId)
-      _ = emailService.sendPaypalThankEmail(payment, capturePaypalPaymentData.acquisitionData.campaignCodes)
+      _ = emailService.sendThankYouEmail(payment.getPayer.getPayerInfo.getEmail)
```
Comment (joelochlann) [inline src/main/scala/backend/PaypalBackend.scala]: oh wait... should we have the same `signedInUserEmail` logic on this one?

and also - `payment.getPayer.getPayerInfo.getEmail` could throw an exception so we need to wrap in `catchNonFatal`

Comment (jbguardian) [reply src/main/scala/backend/PaypalBackend.scala]: 👍

Comment (joelochlann) [reply src/main/scala/backend/PaypalBackend.scala]: have confirmed with Max that the apps are expecting `capture` to accept the signedInUser param

File: src/main/scala/backend/PaypalBackend.scala [hunk 2]
```diff
@@ -95,24 +95,24 @@ class PaypalBackend(
   EitherT[Future, BackendError, Payment] = {
     for {
       payment <- getPaymentFromCapturePaypalPaymentData(capturePaypalPaymentData)
-      identityId <- getOrCreateIdentityIdFromEmail(payment.getPayer.getPayerInfo.getEmail)
+      paymentEmail <- Either.catchNonFatal(payment.getPayer.getPayerInfo.getEmail)
```
Comment (joelochlann) [inline src/main/scala/backend/PaypalBackend.scala]: it's a bit verbose - could you just add a `getEmail` method to the `PayPalService`?

**Pull request merged**